sdk/src/main/java/com/karhoo/sdk/api/model/adyen/PaymentsDetailsRequestPayload.kt

```diff
@@ -0,0 +1,10 @@
+package com.karhoo.sdk.api.model.adyen
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class PaymentsDetailsRequestPayload(
```

Comment:
Are you using this anywhere?

Reply:
Not at the moment, Added it in from the iOS work on a duplicate of this ticket.

I can remove all of the unused models I have added in so far from iOS?